sycl/doc/extensions/experimental/sycl_ext_oneapi_device_image_backend_content.asciidoc

```diff
@@ -101,7 +99,9 @@ class device_image {
   backend ext_oneapi_get_backend() const noexcept;
   std::vector<std::byte> ext_oneapi_get_backend_content() const;
 
-  std::span<std::byte> ext_oneapi_get_backend_content_view() const; // Requires C++20
+#if defined(__cpp_lib_span)
+  std::span<const std::byte> ext_oneapi_get_backend_content_view() const;
+#endif
```

Comment:
This looks like a merge mistake.  The original line (with "Requires C++20") is correct.  You just just add `const` to that line.

Reply:
Yes, thanks!